nodejs/week3/warm-up/index.js

```diff
@@ -0,0 +1,78 @@
+const express = require("express");
+const app = express();
+
+app.use(
+    express.urlencoded({
+        extended: true
+    })
+)
+app.use(express.json());
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.get("/calculator/multiply", (req, res) => {
+    if ("firstParam" in req.query && "secondParam" in req.query) {
```

Comment:
One of the homework requests was "There can be an infinite number of query parameters!"

Reply:
Thankyou Svetlana. However, I tried it with several first and second parameter and it accepts all of them

Reply:
/calculator/add?firstParam=1&secondParam=2&thirdParameter=3
will this work?

Reply:
ohhh ok ... now i get it..I was assuming just infinite number of first and second parameters